packages/components/src/Spinner/BaseSpinner.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Spinner.css";
+
+export interface BaseSpinnerProps {
+  /**
+   * Specifies the size of the spinner
+   *
+   * @default "base"
+   */
+  readonly size?: "small" | "base";
+
+  /**
+   * Spinner becomes an inline element when true,
+   * otherwise it is a block element
+   */
+  readonly inline?: boolean;
```

Comment:
Thinking about this prop and I'm wondering if we could remove this as a property and make it dependent on whether or not `children` are present in the <Spinner>? (This might be a terrible idea, just wanted to discuss it)